plutus-core/plutus-core/src/Codec/Extras/SerialiseViaFlat.hs

```diff
@@ -45,16 +44,16 @@ readDeserialiseFailureInfo (CBOR.DeserialiseFailure byteOffset reason) =
   DeserialiseFailureInfo byteOffset $ interpretReason reason
  where
   -- Note that this is subject to change if `cborg` dependency changes.
-  -- Currently: cborg-0.2.9.0
+  -- Currently: cborg-0.2.10.0
```

Comment:
I don't think I'd looked at this code before.  It'll probably be sheer luck if we notice that the `cborg` error strings change at some time in the future!  I suppose it'd be harmless though.